patroni/dcs/zookeeper.py

```diff
@@ -114,7 +114,7 @@ def set_ttl(self, ttl):
             return True
 
     def set_retry_timeout(self, retry_timeout):
-        self._client._retry.deadline = retry_timeout
+        self._client.retry.deadline = retry_timeout
```

Comment:
Unfortunately this change is incompatible with new kazoo, because KazooClient.retry is a function, which copies a `KazooClient._retry`...

```python
In [1]: import kazoo.client
In [2]: c = kazoo.client.KazooClient()

In [3]: type(c.retry)
Out[3]: function

In [4]: type(c._retry)
Out[4]: kazoo.retry.KazooRetry
```